packages/cloud_functions/cloud_functions_platform_interface/lib/src/method_channel_cloud_functions.dart

```diff
@@ -7,9 +7,11 @@ part of cloud_functions_platform_interface;
 /// [CloudFunctionsPlatform] implementation that delegates to a [MethodChannel].
 class MethodChannelCloudFunctions extends CloudFunctionsPlatform {
   /// The [MethodChannel] to which calls will be delegated.
+  /// Note that this name must match the name used in the platform (iOS/Android)
+  /// plugin.
   @visibleForTesting
   static const MethodChannel channel = MethodChannel(
-    'plugins.flutter.io/cloud_functions',
```

Comment:
When I created this file, I was copying and modifying a different plugin and didn't understand exactly how `MethodChannel` names worked. The plugin I was cribbing from used a different naming convention, and I didn't catch the mistake. The unit tests only check to see that the `MethodChannel` gets invoked, not what the channel name is. Moreover, the actual iOS and Android implementations are in the `cloud_functions` plugin, not this platform interface plugin, meaning integration testing is problematic. This feels to me like an architecture problem -- like maybe the method channel implementation should be in the plugin with the platform implementations -- but there are ramifications that get complicated, and the "right thing" might wind up being a breaking change.